_pages/filing-a-complaint.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Filing a Complaint
+description: "The Americans with Disabilities Act provides an important tool to fight discrimination: filing a complaint with an appropriate federal agency.  This page outlines the steps to get you started."
+permalink: /filing-a-complaint/
+lead: |-
+  The Americans with Disabilities Act provides an important tool to fight discrimination:  filing a complaint with an appropriate federal agency.  This page outlines the steps to get you started.
+---
+
+## Who You Can File a Complaint Against
+
+If you believe that you or someone else was discriminated against based on a disability, you can file an Americans with Disabilities Act (ADA) complaint against:
+
+- a state government or local government, such as:
+  - public hospitals
+  - public schools
+  - other state or local government programs
+
+- a private business that serves the public, such as a:
+  - restaurant
+  - doctor's office
+  - retail store
+  - hotel
+
+## Where to File Your Complaint
+
+The nature of your complaint determines where you should file it. The table below tells you where to file your complaint.
+
+| Type of Complaint                                              | Where to file online                                                                                                                              |
```

Comment:
Since these are table headers, I think they should both be title case in accordance with the DRS style guide. So `Type of Complaint` and `Where to File Online`.